.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Please fill out this information before submitting a bug:
+
+  - `ruby --version`: ...
+  - `gem --version`: ...
```

Comment:
Yes, `gem env` would definitely be more useful, but it'd also make the issues be incredibly long. Maybe asking for that and whoever does triaging can move it to a gist if it's not already pastebinned somewhere? idk.